client/src/containers/Register/styles.css

```diff
@@ -0,0 +1,68 @@
+.register-container {
+    text-align: center;
+}
+
+.register-container .form_element {
+    margin-top: 20px;
+}
+
+.register-container h2 {
+    font-size: 33px;
+    font-weight: 300;
+    color: #02bcd4;
+}
+
+.register-container input {
+    font-size: 20px;
+    border-radius: 3px;
+    border: 1px solid #ebebeb;
+    box-sizing: border-box;
+    padding: 11px 31px;
+    background: #f7f7f7;
+    font-weight: 300;
+    width: 260px;
+}
+
+.register-container button {
+    font-size: 20px;
+    padding: 10px 52px;
+    background: #02bcd4;
+    color: #fff;
+    font-weight: 300;
+    border-radius: 3px;
+    border: none;
+    margin-top: 20px;
+    width: 260px;
+}
+
+.register-container .error {
```

Comment:
Error message looks a little bit huge.
I propose you to change styles:
```
.register-container .error {
    margin-top: 5px;
    font-size: 13px;
    font-weight: 400;
}
```
It is not mistake and depend on you wishes.

Reply:
Thanks! I will try your proposition! :)

Reply:
Finally, I changed the margin and weight, but not the size... I think 13px is too small... :D 
Thank you, now it looks much better! 👍